src/components/include/connection_handler/connection_handler_observer.h

```diff
@@ -83,15 +85,18 @@ class ConnectionHandlerObserver {
   /**
    * \brief Callback function used by connection_handler
    * when Mobile Application initiates start of new service.
+   * Result must be notified through NotifyServiceStartedResult().
    * \param deviceHandle Device identifier within which session has to be
    * started.
    * \param sessionKey Key of started session.
    * \param type Established service type
+   * \param params Configuration parameters for this service
    */
-  virtual bool OnServiceStartedCallback(
+  virtual void OnServiceStartedCallback(
```

Comment:
Sure, done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/9f0795170b9cc46c6c8565ba967da6ae3bdbc613.